docs/howto/smart_contract_upload.md

```diff
@@ -0,0 +1,138 @@
+# Smart Contract Upload
+
+This tutorial covers how to create a circuit on a splinter network between 
+two nodes and upload a smart contract via the circuit.
+
+## Prerequisites
+
+* A splinter network with at least two nodes running. The tutorial will refer
+  to the two nodes as "alpha" and "beta" respectively
+* The scabbard command line tool installed with experimental features enabled
+* The splinter command line tool installed with experimental features enabled
+* A compiled smart contract. Check [here](https://sawtooth.hyperledger.org/docs/sabre/nightly/master/application_developer_guide.html#) for details on how to create and compile a smart contract
+
+## Procedure
+
+1. Create a new circuit
+
+    a. Propose the circuit on alpha node
+
+    ```
+    $ splinter circuit create \
+        --url http://splinterd-alpha:8085 \
+        --key <path_to_alpha_private_key> \
+        --node alpha-node-000::tls://splinterd-alpha:8044 \
+        --node beta-node-000::tls://splinterd-beta:8044 \
+        --service scabbard-service-alpha::alpha-node-000,beta-node-000
+        --service scabbard-service-beta::beta-node-000,alpha-node-000
+        --service-peer-group beta-node-000,alpha-node-000 \
+        --service-arg *::admin_keys=<alpha_node_pub_key> \
+        --service-arg scabbard-service-beta::peer_services=alpha-node-000 \
+        --service-arg scabbard-service-alpha::peer_services=beta-node-000 \
+        --service-type *::scabbard \
+        --management tutorial \
+        --auth-type trust
+    ```
+    
+    The `--service-arg`s supplied are specific to the scabbard service and are
+    required in order to use and configue smart contracts. `admin_keys`
+    identify who is allowed to add permissions to a contract and `peer_services`
+    is for identifying which parties are needed for consensus.
+    
+
+    b. Vote to accept the circuit on node beta 
+    
+    ```
+    $ splinter circuit vote \
+        --key <path_to_beta_private_key> \
+        --url http://splinterd-beta:8085 my-circuit --accept
```

Comment:
my-circuit should be <circuit id>. Also it would be helpful to add a step to check the proposals so you can see what the circuit id is. (since it is generated by the cli know)